modules/doc/content/getting_started/installation/install_mfem.md

```diff
@@ -0,0 +1,64 @@
+# Installing MFEM-MOOSE
+
+To enable `MFEM-MOOSE` capabilities, it is necessary to install all of its dependencies, including `MFEM` itself and `Conduit`. You may do so by performing the following steps:
+
+1. After cloning the `MOOSE` repository, navigate to the root directory and run the following commands:
+
+```bash
+export METHOD=opt
+export MOOSE_JOBS=10
+```
+
+First, we need to build `PETSc`. To do that, we can run the `PETSc` installer script:
+
+```bash
+./scripts/update_and_rebuild_petsc.sh [options]
+```
+
+You may substitute `[options]` by any `PETSc` configure flags, which are discussed in the [`PETSc` install page](https://petsc.org/release/install/install). If you intend to build `MFEM-MOOSE` with GPU offloading capabilities, make sure to include `--with-cuda` or `--with-hip` to the flags. If desired, you may also specify the GPU architecture with `--with-cuda-arch=[arch]` or `--with-hip-arch=[arch]`.
+
+!alert note
+The `PETSc` GPU architecture specification takes in only the architecture number for `CUDA` builds, for instance for the `sm_80` architecture you would add the flag `--with-cuda-arch=80`. For `HIP` builds you should use the entire label, for instance `--with-hip-arch=gfx908`.
+
+
+Next, we build `libMesh`, `Conduit`, and `Wasp` by running their respective scripts:
+
+```bash
+./scripts/update_and_rebuild_libmesh.sh
+./scripts/update_and_rebuild_conduit.sh
+./scripts/update_and_rebuild_wasp.sh
+```
+
+We must then build `MFEM` itself by running its script:
+
+```bash
+./scripts/update_and_rebuild_mfem.sh [options]
+```
+
+Here, in `[options]`, you may wish to include `MFEM`'s cmake options, which are detailed [in this page](https://deepwiki.com/mfem/mfem/8-build-system-and-development). Notably, like with the `PETSc` install, if you are building `MOOSE-MFEM` with GPU offloading capabilities, here your flags should include `-DMFEM_USE_CUDA=YES` or `-DMFEM_USE_HIP=YES`. For the GPU architecture specification, use `-DCUDA_ARCH=[arch]` or `-DHIP_ARCH=[arch]`.
```

Comment:
You need to mention passing an argument to trust the superlu_dist version when using cuda (and I'm guessing for hip as well)